docs/src/selectors.md

```diff
@@ -762,6 +762,45 @@ await page.Locator("button:near(.promo-card)").ClickAsync();
 All layout selectors support optional maximum pixel distance as the last argument. For example
 `button:near(:text("Username"), 120)` matches a button that is at most 120 pixels away from the element with the text "Username".
 
+## Selecting elements by label text
+
+Targeted input actions in Playwright automatically distinguish between labels and controls, so you can target the label to perform an action on the associated control.
+
+For example, consider the following DOM structure: `<label for=password>Password:</label><input id=password type=password>`. You can target the label with something like `text=Password` and perform the following actions on the input instead:
```

Comment:
nit: I'd personally prefer using quotes for the attributes.

Reply:
Good point!